examples/simple/simple_test.go

```diff
@@ -96,6 +115,7 @@ func Example() {
 
 	// Read all invites.
 	invitations, _ := invitationRepository.FindAll()
+	assert.Equal(s.T(), 3, len(invitations))
```

Comment:
The example should contain no asserts, instead it should be easy to read. More like a program a user would write. It will however check the output against the ending comment. It already asserts the length by comparing to the three printed lines in the output.